concent_api/core/templatetags/admin_tags.py

```diff
@@ -0,0 +1,52 @@
+import os
+from django import template
+from constance import config
+from ..models import Subtask
+
+register = template.Library()
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+
+@register.assignment_tag
+def base_dir_path():
+    return BASE_DIR
```

Comment:
I think that settings values are already available in templates, but I am not sure, please check in docs. If not keep it but use value from `django.conf.settings`.

Reply:
Done.